src/plugins/irc-events/error.js

```diff
@@ -40,4 +40,19 @@ module.exports = function(irc, network) {
 		var random = irc.user.nick + Math.floor(10 + (Math.random() * 89));
 		irc.changeNick(random);
 	});
+
+	irc.on("nick invalid", function(data) {
+		var lobby = network.channels[0];
+		var msg = new Msg({
+			type: Msg.Type.ERROR,
+			text: "Nickname " + data.nick + " is invalid: " + data.reason,
+		});
+		client.emit("msg", {
+			chan: lobby.id,
+			msg: msg
+		});
+
+		var random = "i" + Math.random().toString(36).substr(2, 10); // 'i' so it never begins with a number
```

Comment:
It could indeed, but we could either default another invalid nick to `"lounge-user"+number`, or assume that the configuration file is sane as we do for other defaults, such as booleans, etc.